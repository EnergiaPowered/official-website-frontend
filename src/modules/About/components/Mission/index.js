import React from "react";

/**
 * Component which contain The Mission of Energia Powered
 * 
 * @component 
 * @returns {JSX} Return the Mission Component
 */

export default () => {
  return (
    <section className="bg-section about-mission component-font">
      <h2 className="section-title">Our Mission</h2>
      <p>
        Energia Powered attempts to deliver values and achieve its vision
        through different projects:
      </p>

      <ul>
        <li> Conference </li>
        <li> Training Sessions </li>
        <li> Events </li>
        <li> Field Trips </li>
      </ul>
    </section>
  );
};
