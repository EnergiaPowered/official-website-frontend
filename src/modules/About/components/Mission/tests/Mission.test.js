import React from "react";

import Mission from "../index";

import { create } from "react-test-renderer";
import { cleanup } from "@testing-library/react";

afterEach(cleanup);

// matches snapshot
it("matches snapshot", () => {
  const component = create(<Mission />);

  expect(component.toJSON()).toMatchSnapshot();
});
