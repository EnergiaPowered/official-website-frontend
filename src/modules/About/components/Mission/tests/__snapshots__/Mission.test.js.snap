// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<section
  className="bg-section about-mission component-font"
>
  <h2
    className="section-title"
  >
    Our Mission
  </h2>
  <p>
    Energia Powered attempts to deliver values and achieve its vision through different projects:
  </p>
  <ul>
    <li>
       Conference 
    </li>
    <li>
       Training Sessions 
    </li>
    <li>
       Events 
    </li>
    <li>
       Field Trips 
    </li>
  </ul>
</section>
`;
